src/status_im/ui/screens/wallet/settings/events.cljs

```diff
@@ -0,0 +1,18 @@
+(ns status-im.ui.screens.wallet.settings.events
+  (:require [status-im.utils.ethereum.core :as ethereum]
+            [status-im.utils.handlers :as handlers]))
+
+(defn- mark-checked [ids id checked?]
```

Comment:
is this like `toggle-checked`? otherwise from name i would expect that something will be always checked instead of be changed in both ways

Reply:
Right.